### PR TITLE
updating the PAF doc that serena started

### PR DESCRIPTION
**Pull request by Ninicorn**

### CODE REVIEW

File: docs/source/model_design/risk_factors.rst [hunk 1]
```diff
@@ -531,12 +531,99 @@ Difference measures relating to the whole population tend to be more useful and
 Population Attributable Fraction (PAF)
 ++++++++++++++++++++++++++++++++++++++
 
-The population attributable fraction is a quantification of the proportion of 
-of a given cause outcome, such as cases, deaths, or DALYs, that could be 
-eliminated by removing a risk exposure.
+Measures of population impact estimate the expected impact on a population of removing or changing the distribution of risk factors in that population. In GBD, this is the level of exposure that is equal to the TMREL. They take into account the both the strength of the association (estimated by a measure of effect, like the rate ratio) **and** the distribution of the risk factor in the population. It assumes that we have established that the association between disease and risk factor is *causal*. If this assumption is true, population impact estimates measure how much of the disease in the population is caused by the suspected risk factor.
```
Comment (aflaxman) [inline docs/source/model_design/risk_factors.rst:534]: second sentence is unclear, what is the level of exposure?  fourth paragraph is unclear, what is "it" that assumes this?

File: docs/source/model_design/risk_factors.rst [hunk 2]
```diff
@@ -531,12 +531,99 @@ Difference measures relating to the whole population tend to be more useful and
...
+
+Intuitively, PAF equals (O − E)/O, where O and E refer to the observed number of cases and the expected number of cases under no exposure, respectively. The term “attributable” has a causal interpretation: PAF is the estimated fraction of all cases that would not have occurred if there had been no exposure. As an example, in early 1950, Doll derived O = 11189 and E= 1875 using the Doll and Hill case-control study of smoking and lung cancer deaths throughout England and Wales, so the smoking PAF for lung cancer deaths was (11189 − 1875)/11189= 83%.
```
Comment (aflaxman) [inline docs/source/model_design/risk_factors.rst:536]: define PAF before using acronym. "E" should be the number expected for TMRED, which is not always no exposure.  Perhaps lead with the Doll example, and then generalize.

File: docs/source/model_design/risk_factors.rst [hunk 3]
```diff
@@ -531,12 +531,99 @@ Difference measures relating to the whole population tend to be more useful and
...
+
+It is important to remember that measures of population impact are **specific to the population studied**, and can **only be generalised to populations with exactly the same distribution of risk factors**. Also note that risk factors that are strongly associations but which are rare, like being exposed to an X ray in pregnancy and leukaemia in childhood, may have a large measure of effect but small measure of impact.
```
Comment (aflaxman) [inline docs/source/model_design/risk_factors.rst:538]: X-ray (with a dash, not a space)

File: docs/source/model_design/risk_factors.rst [hunk 4]
```diff
@@ -531,12 +531,99 @@ Difference measures relating to the whole population tend to be more useful and
...
++----------------+--------------+--------------+---------------------+ 
+
+
+The PAR is the absolute difference between the risk/rate in the whole population (r) and the risk/rate in the unexposed group (r0). 
+
+Population attributable risk (PAR) is calculated as
+
+	PAR = r - r0
+
+	*Note* that the risk difference (RD) in the earlier section contrasts the rate/risk in the exposed group (r1) and the rate/risk in the unexposed group (r0 = r1-r0.
```
Comment (aflaxman) [inline docs/source/model_design/risk_factors.rst:566]: missing paren at end of line

File: docs/source/model_design/risk_factors.rst [hunk 5]
```diff
@@ -531,12 +531,99 @@ Difference measures relating to the whole population tend to be more useful and
...
+
+The population attributable fraction is a quantification of the proportion of a given cause outcome, such as cases, deaths, or DALYs, that could be eliminated by removing a risk exposure.
+It is the proportion of all cases in the whole study population (exposed and unexposed) that may be attributed to the exposure, assuming a causal association.
+The population attributable risk fraction (PAF) is estimated by dividing the population attributable risk by the risk in the total population (r).
+
+   PAF = PAR/r
+
+       = (r - r0) / r
+
+When only the risk ratio (RR) and the **prevalence of exposure in the population** are known, PAF can also be written as:
```
Comment (aflaxman) [inline docs/source/model_design/risk_factors.rst:602]: might as well define RR here, too, in terms of your 2x2 table

File: docs/source/model_design/risk_factors.rst (hunk 1 above)
Comment (Ninicorn) [reply docs/source/model_design/risk_factors.rst:534]: edited for clarity

File: docs/source/model_design/risk_factors.rst (hunk 2 above)
Comment (Ninicorn) [reply docs/source/model_design/risk_factors.rst:536]: ok!

**Pull request merged**